### ISSUE TITLE
Part of speech of 'ᠪᡝ'

### ISSUE THREAD

**Issue by ftyers** [hohojo/UD_Xibe-IU#3]
In the following sentence, the word `ᠪᡝ` is tagged with the `PART` tag. I think that absent of any indications to the contrary, it would be better to tag it with `ADP`, given its syntactic function. The number of tokens tagged with `PART` should be very limited, and explicit.

```
# sent_id = grammarbook_sjo_p1_1
# sent = ᡧᡠᠨ ᡨᡠᠴᡞᡫᡞ , ᡫᠠᠷᡥᡡᠨ ᡩᠣᠪᠣᠷᡞ ᠪᡝ ᠪᠣᡧᠣᡥᠣ 。
# sent[phon] = šun tucifi , farhvn dobori be bošoho.
# sent[eng] = The sun rises, drives the dark away.
1	ᡧᡠᠨ	ᡧᡠᠨ	NOUN	_	Case=Nom	2	nsubj	_	Translit=šun
2	ᡨᡠᠴᡞᡫᡞ	ᡨᡠᠴᡞᠮᠪᡞ	VERB	_	Tense=Pres|VerbForm=Conv	0	root	_	Translit=tucifi
3	,	,	PUNCT	_	_	_	2	punct	Translit=,
4	ᡫᠠᠷᡥᡡᠨ	ᡫᠠᠷᡥᡡᠨ	ADJ	_	_	5	amod	_	Translit=farhvn
5	ᡩᠣᠪᠣᠷᡞ	ᡩᠣᠪᠣᠷᡞ	NOUN	_	_	7	dobj	_	Translit=dobori
6	ᠪᡝ	ᠪᡝ	PART	_	Case=Acc	5	case	_	Translit=be
7	ᠪᠣᡧᠣᡥᠣ	ᠪᠣᡧᠣᠮᠪᡞ	VERB	_	Tense=Past	_	2	conj	Translit=bošoho
8	。	。	PUNCT	_	_	_	7	punct	Translit=.
```

@jonorthwash what do you think?

**Comment by hzh1-cl** [comment]
Xibe nouns have 7 cases, and they are almost all used in isolation except when the word is monosyllabic, ex: yaci (to where) in which ya is 'where', '-ci' is case marker. So all the isolated case markers are tagged ADP rather than PART?

**Comment by ftyers** [comment]
That would be my suggestion, [`ADP`](https://universaldependencies.org/u/pos/ADP.html) has a syntactic function, "a closed set of items that occur before (preposition) or after (postposition) a complement composed of a noun phrase, noun, pronoun, or clause that functions as a noun phrase, and that form a single structure with the complement to express its grammatical and semantic relation to another unit within a clause."

The category [`PART`](https://universaldependencies.org/u/pos/PART.html) is vague and generally avoided, "In general, the `PART` tag should be used restrictively and only when no other tag is possible".

**Comment by jonorthwash** [comment]
Yes, I agree.  The case suffixes/clitics/"particles" should almost definitely be `ADP`.
